modules/base/variables.tf

```diff
@@ -49,3 +49,9 @@ variable "subscription_id" {
   description = "Azure Subscription (ID) to use"
   type        = string
 }
+
+variable "humanitec_org_id" {
+  description = "Humanitec Organization ID (required for Backstage)"
+  type        = string
+  default     = null
```

Comment:
Isn‘t this required in any case? I think in this case the „default = null“ should be dropped as well as the „required for Backstage“

Reply:
Fixed!